### PR TITLE
Add WikiProject aspect

### PR DESCRIPTION
**Pull request by carlinmack**
Created a new aspect for WikiProjects. Has panels for items maintained by the WikiProject and items on the focus list of the WikiProject

image

I have also added a new function `get_prop_value` to `query.py`, which for a given QID and property, returns the value of the (Q, property, value) (where the value is also a QID. This is used to return the main subject.

### Caveats

> Please list anything which has been left out of this PR or which should be considered before this PR is accepted
Check any of the following which apply:

* [ ]  Breaking change (fix or feature that would cause existing functionality to not work as expected)
* [ ]  This change requires a documentation update
    * [ ]  I have made corresponding changes to the documentation
* [ ]  This change requires new dependencies (please list)

*If you make changes to the Python code*
  
* [x]  My code passes the [tox](https://tox.readthedocs.io/en/latest/) check, I can receive warnings about tests, documentation or both

### Testing
Please describe the [tests](https://numpy.org/doc/stable/reference/testing.html) that you ran to verify your changes. Provide instructions, so we can reproduce. Please also list any relevant details for your test configuration.

* If there are no items maintained by the [wikiproject](https://www.wikidata.org/wiki/Q18437553):

image

### Checklist
* [x] I have [commented](https://peps.python.org/pep-0008/#comments) my code, particularly in hard-to-understand areas
* [x] My changes generate no new warnings
* [x] I have not used code from external sources without attribution
* [x] I have considered [accessibility](https://www.w3.org/standards/webdesign/accessibility) in my implementation 
* [x] There are no remaining debug statements (print, console.log, ...)

CC @Daniel-Mietchen

### CODE REVIEW

**Comment by carlinmack** [comment]
Will consider adding panels suggested here https://github.com/WDscholia/scholia/issues/1853

**Comment by Daniel-Mietchen** [comment]
> Will consider adding panels suggested here #1853

Yes, please do. Note that the second panel would give better results for P6104 instead of P5008, as per [this comment](https://github.com/WDscholia/scholia/issues/1853#issuecomment-1593184563).

**Comment by carlinmack** [comment]
Daniel, if you look at the image you will see that I already use both P6104 and P5008 :)

**Comment by carlinmack** [comment]
Once Shex schemas are queryable via SPARQL ([coming soon](https://www.wikidata.org/wiki/Wikidata_talk:Schemas#New_EntitySchema_data_type_is_ready_for_testing_on_Test_Wikidata)) I'll add a panel which shows the coverage of the schema properties for the relevant items.

**Comment by carlinmack** [comment]
Added panels from a selection of the panels of the topic aspect:

image
image

File: scholia/query.py [hunk 1]
```diff
@@ -295,6 +295,54 @@ def identifier_to_qs(property, identifier):
...
+        Property
+
+    Returns
+    -------
+    v : str
+        qid of value
+
+    Notes
+    -----
+    The Wikidata Query Service is queries to resolve the given identifier. If
```
Comment (fnielsen) [inline scholia/query.py:315]: queries -> queried

File: scholia/app/templates/wikiproject.html [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+
+var url = 'https://www.wikidata.org/w/api.php?action=wbgetentities&ids=' +
+    '{{ main_subject }}' +
+    '&format=json&callback=?';
+  
+$.getJSON(url, function (data) {
+  var item = data.entities["{{ main_subject }}"];
+  if ('en' in item.labels) {
+    $("#main-subject").append(': <a href="/topic/{{ main_subject }}">' + item.labels.en.value + '</a>');
```
Comment (fnielsen) [inline scholia/app/templates/wikiproject.html:30]: Do we need an escape here for item.labels.en.value?

**Comment by carlinmack** [comment]
Had a call with Daniel where we worked on this aspect and he suggested using P6104/P5008 maintained by/focus list rather than getting the P921 main subject for the item. I think I'll remove get_prop_val in this case

**Comment by fnielsen** [comment]
Let me know when you are finished.

**Comment by carlinmack** [comment]
Removed the unused function, now ready to merge :) @fnielsen

**Comment by fnielsen** [comment]
https://scholia.toolforge.org/wikiproject/Q56241615

**Pull request merged**